Assets/Scripts/MapGen/MappableTileset.cs

```diff
@@ -57,6 +57,12 @@ public Vector2 GetCenter()
             return transform.position + bounds.center * scale;
         }
 
+        public float GetUnscaleValue()
+        {
+            float scale = Mathf.Max(transform.localScale.x, transform.localScale.y);
+            return scale * 0.5f;
```

Comment:
`0.5f` is a magic number so far used quite a bit across the codebase. May make sense to make this a constant.

Reply:
For now could we just add it as a const on top of this file ?